2-mandatory/2-financial-times.js

```diff
@@ -14,24 +20,54 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    
+    let shortestLine;
+    let length_headline = 100000;
+
+    for (headline of allArticleTitles) {
+        let currentTitleWordCount = headline.split (" ").length;
+        
+     if (currentTitleWordCount < length_headline) {
+        shortestLine = headline;
+        length_headline = currentTitleWordCount;
+     }
+    }
+        
+    return shortestLine;
+    
 }
```

Comment:
Comment:  I see you use 100000; as a value. I assume it is a made up value. 

Consider: Have a look at how you can use [i] It is not used in any of your coding. 

Example:
function titleWithFewestWords(allArticleTitles) {
    let shortestTitle = allArticleTitles[0];
    
    for (let i = 1; i < allArticleTitles.length; i++) {
        let currentTitle = allArticleTitles[i];
        if (currentTitle.length < shortestTitle.length) {
            shortestTitle = currentTitle  
        }
    }
    return shortestTitle;
}